Charles_Math.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Charles_Maths</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script src="script.js"></script>
+</head>
+<body>
+    <!--Maths operation Assignment--> 
+    <p style="color:rgb(248, 72, 72);"> *Please, kindly view the console onclick of the submit button bellow ro view the script Function Message*</p>
+    <input type="submit" onclick="displayMessage()">
```

Comment:
I don't see where the "displayMessage" function is set up.  You should have the function code placed in script.js.